torch/testing/_asserts.py

```diff
@@ -649,8 +707,7 @@ def assert_equal(
     Raises:
         UsageError: If an array-or-scalar-like pair has different types.
         UsageError: If a :class:`torch.Tensor` can't be constructed from an array-or-scalar-like.
-        UsageError: If any tensor is quantized or sparse. This is a temporary restriction and will be relaxed in the
-            future.
+        UsageError: If any tensor is sparse. This is a temporary restriction and will be relaxed in the future.
```

Comment:
Should we just document assertEqual as equivalent to `partial(assert_close(atol=0, rtol=0))`?

Reply:
See https://github.com/pytorch/pytorch/pull/58918. Although I'm not sure anymore if we should have an `assert_equal` at all. With all the new keyword parameters that might be added, it will be a pain to keep the docstrings in sync. Maybe we can use `assert_equal` as an example of how `assert_close` can be `functools.partial`'ed?